server/src/main.ts

```diff
@@ -24,10 +25,11 @@ async function bootstrap() {
 
     const config = new DocumentBuilder()
         .setTitle('Momentum Mod API')
-        .setDescription('The Momentum Mod API - Made with 💖')
+        .setDescription('The Momentum Mod API - Made with 🦵')
```

Comment:
Made with leg emoji?

Reply:
Fuck, I was hoping this wouldn't be noticed :( 
I would rather we remove this than have it at all, it's meaningless and corny

Reply:
Nuked it